variables.tf

```diff
@@ -8,9 +8,9 @@ variable "resource_group_id" {
 }
 
 variable "region" {
-  description = "Region to provision COS bucket. Also used when creating Key Protect / Key Protect Keys for encryption. NOTE: If 'var.encryption_enabled' is true and an existing Key Protect instance is passed in using 'var.existing_key_protect_instance_guid', this must be the region of the existing Key Protect instance."
+  description = "The region to provision the bucket. If you pass a value for this, do not pass one for var.cross_region_location."
   type        = string
-  default     = "us-south"
+  default     = null
```

Comment:
I think we should still default this to a region so the default behaviour will be to deploy a single region bucket )otherwise its a breaking change for current consumers)